site/vite.config.ts

```diff
@@ -0,0 +1,37 @@
+import react from "@vitejs/plugin-react"
+import path from "path"
+import { defineConfig } from "vite"
+
+export default defineConfig({
+  plugins: [react()],
+  publicDir: path.resolve(__dirname, "./static"),
+  build: {
+    outDir: path.resolve(__dirname, "./out"),
+  },
+  define: {
+    "process.env": process.env,
+  },
+  server: {
+    port: process.env.PORT ? Number(process.env.PORT) : 8080,
+    proxy: {
+      "/api": {
+        target: process.env.CODER_HOST || "http://localhost:3000",
+        ws: true,
+        secure: process.env.NODE_ENV === "production",
+      },
+    },
+  },
+  resolve: {
```

Comment:
woah does this make it so we can do shorter imports or something?

Reply:
Yes, unfortunately, looks like Vite is not smart enough to apply the TS aliases automatically.